### ISSUE TITLE
ability to process very large msgs

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#622]
https://mail.google.com/mail/u/flowcrypt.compatibility@gmail.com/#inbox/WhctKHTKkvbLlQBkHzWnLXjcktLCnqgSRkCQRvJmrdQCHClFGCJPssqnrVdDNWdTbDdhFjQ

https://mail.google.com/mail/u/flowcrypt.compatibility@gmail.com/#inbox/WhctKHTKkvbLlHscRBsbkmjcbRZvGpDFlNgVMJnWzmfNMbWNhBfqPRcxwFLMbnvcpqZtbPV

depends on: #609 #610 #585 #580

**Comment by tomholub** [comment]
I think we already finished this, but please test the two emails mentioned. 

If it works well, you could even add them both as part of the automated tests?

**Comment by DenBond7** [comment]
> I think we already finished this, but please test the two emails mentioned.

Yes, it works well. It'll be great to do https://github.com/FlowCrypt/flowcrypt-android/issues/608. This issue is very useful for large messages.

> you could even add them both as part of the automated tests?

ok, I'll add

**Comment by DenBond7** [comment]
I've added a test for decryption a large message. But I'll complete testing of loading big messages only after #646. I've created a separate issue for that. For now, I'm closing the current issue.
